### PR TITLE
User authentication

### PR DESCRIPTION
**Pull request by malwas**
User authentication as been added using Device gem

### CODE REVIEW

**Comment by siepet** [comment]
We have empty .coffee file that we no longer want or need - could you remove it?
We also don't need `test/` files, could you remove them as well?

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (bochen421) [inline app/models/user.rb]: We don't include :confirmable module for users here so we should either change it here or delete columns for confirmable module from migration.

File: db/migrate/20180526082927_devise_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      t.datetime :locked_at
+
+
+      t.timestamps null: false
+    end
+
+    add_index :users, :email,                unique: true
+    add_index :users, :reset_password_token, unique: true
+    # add_index :models, :confirmation_token,   unique: true
+    # add_index :models, :unlock_token,         unique: true
```
[the remark this answers is not in the document]
Comment (malwas) [reply db/migrate/20180526082927_devise_create_users.rb]: done :)

File: app/models/user.rb (hunk 1 above)
Comment (malwas) [reply app/models/user.rb]: done

**Pull request merged**